parsexml.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+#Author:        Sasan Bahadaran
+#Date:          2/29/16
+#Organization:  Commerce Data Service
+#Description:   This script crawls the files directory and finds the metadata
+#xml file for each set of pdf files.  It then gets the document id from
+#the metadata and finds the associated extracted pdf text and adds it to the
+#structure of the document, then transforms the document to json and saves it
+#to a file.
+
+import json, xmltodict, os, logging, time
+
+#get value for specified tag from a list
+def getData(list,tag):
+    value = list[tag]
+
+    return value
+
+#add data with specified tag to existing list
+def setData(list,tag,text):
+    list[tag] = text
+
+#get specified folder from folder path
+def getFolder(root,level):
+    folder = root.split(os.path.sep)[level]
+
+    return folder
+
+if __name__ == '__main__':
+    logging.basicConfig(filename='logs/parse-xml-log-'+time.strftime('%Y%m%d-%H%M%S'),\
+    level=logging.INFO, format='%(asctime)s - %(name)s - %(levelname)s\
+    -%(message)s',datefmt='%Y%m%d %H:%M:%S')
+ 
+    logging.info("-- [JOB START]  ----------------")
+    logging.info("-- =============== Script starting ====================")
+
+    #crawl through each main directory and find the metadata xml file
+    for root, dirs, files in os.walk('files/'):
+        for name in files:
+            if name.endswith('.xml'):
+                with open(os.path.join(root,name)) as fd:
```

Comment:
All these cases of `join(root,name)` and similar should be able to be replaced with `os.path.abspath(name)`.